src/components/dialogs/dialogAlert/dialogAlertRoot/dialogAlertRoot.tsx

```diff
@@ -0,0 +1,83 @@
+import { Content, Overlay, Portal, Root } from '@radix-ui/react-alert-dialog';
+import classNames from 'classnames';
+import type React from 'react';
+import { createContext, useMemo, type ReactNode } from 'react';
+
+export type DialogAlertVariant = 'critical' | 'info' | 'success' | 'warning';
+
+export interface IDialogAlertRootProps {
+    children?: ReactNode;
```

Comment:
I know, pretty obvious 😅 
```suggestion
    /**
     * Children of the component.
     */
    children?: ReactNode;
```